pmultiqc/modules/common/ms/mzid.py

```
@@ -0,0 +1,201 @@
+        for file_path in self.file_paths:
+
+            self.log.info(
+                "{}: Parsing MzIdentML file {}...".format(
+                    datetime.now().strftime("%H:%M:%S"), file_path
+                )
+            )
+
+            mzid_data = mzid.MzIdentML(file_path)
+
+            if len(mzid_data) == 0:
+                raise ValueError("Please check your MzIdentML", file_path)
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)